JsonSchema.Tests/JsonSchema.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
+		<TargetFrameworks>netcoreapp3.1;net8</TargetFrameworks>
```

Comment:
I'd actually prefer dropping 3.1 and having the current LTS versions here (6 & 8).

Also, can you update the other test projects, please?

Reply:
Sure. So you'd like these projects that were `netcoreapp3.1;net6.0` to be `net6.0;net8`?

Reply:
And anything that is netcoreapp3.1 should be net6;net8 too?

Reply:
Should only be tests that have .Net Core 3.1, but yes.

Reply:
As long as we are continuing to support netstandard2.0 then we should continue to have netcoreapp3.1-based tests so that we can exercise both the reflection-based and JsonTypeInfo-based paths. We can have tests built and run 3x if you want to exercise netcoreapp3.1 _and_ net6 and net8.